.studio/applications-service

```diff
@@ -66,6 +66,87 @@ function applications_get_service_groups_unknown() {
       }'
 }
 
+# Run benchmarks for the applications-service
+document "applications_benchmarks" <<DOC
+  Runs benchmarks for the applications-service.
+DOC
+function applications_benchmarks() {
+  check_if_deployinate_started || return 1
+
+  (
+    enter_go_workspace
+    install_if_missing core/go go
+    install_if_missing core/git git
+
+    local tmp_txt=/tmp/new.txt
+    local results_txt=/src/benchmarks/applications-service/txt/new.txt
+    local results_html=/src/benchmarks/applications-service/html/new.html
+
+    log_line "Running applications-service benchmark. (temporarily stored at $(yellow $tmp_txt))"
+    log_line "$(yellow "This might take a while...")"
+    cd components/applications-service || return 1
+    A2_SVC_NAME="applications-service" A2_SVC_PATH="/hab/svc/applications-service" go test \
+      ./... \
+      -bench=Benchmark \
+      -benchmem \
+      -count=5 2>&1 | tee $tmp_txt
+
+    log_line "Saving benchmark results at $(green $results_txt)"
+    grep Benchmark $tmp_txt > $results_txt
+    rm $tmp_txt
```

Comment:
could have just done
```
| tee $results_txt
```
above, I think. benchstat will ignore the non-`Benchmark` lines

Reply:
Ill test this!!! That would be super awesome!!